components/Astronaout.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Lottie from 'react-lottie';
+import animationData from '../ lotties/astronaut.json';
+
+class AstronautLottie extends Component {
+  render() {
+    const defaultOptions = {
+      loop: true,
+      autoplay: true,
+      animationData: animationData,
+      rendererSettings: {
+        preserveAspectRatio: 'xMidYMid slice',
+      },
+    };
```

Comment:
Const data can be kept outside of the object, this is an optimisation as it prevents the data being duplicated for every use.